### ISSUE TITLE
Shows incorrect cursor icon while dragging

### ISSUE THREAD

**Issue by FichteFoll** [easymodo/qimgv#421]
I noticed that the default action of dragging is to move even though it shows the icon with a + (which usually means copying).

In addition to that, it would be cool if I could modify the file operation of the drag and drop action with a modifier key like in other file managing apps. Nautilus uses the following modifiers:

- shift to move
- ctrl to copy
- ctrl+shift to link
- alt to ask

Initially raised in #409.

**Comment by easymodo** [comment]
Yes this will be improved
I want to do a popup similar to what dolphin does

![image](https://user-images.githubusercontent.com/5484275/175769039-7cf24524-90a1-4490-9040-bc8e331a9c52.png)

**Comment by FichteFoll** [comment]
Yes, that looks like the Nautilus alt behavior. Do you want to make ask the default? I think keeping moving as the default is reasonable as that should be expected for most file management operations, at least it is for file managers afaik.
